tests/specs/fiber_context.js

```diff
@@ -0,0 +1,73 @@
+describe('Fiber based context', function () {
+  var fiberServer = meteor({fiber: true});
```

Comment:
Actually I am thinking about encouraging users to declare their favorite `flavor` explicitly because we may change the default behavior in the future.